net-dns/dnssec-validator/dnssec-validator-2.2.3.ebuild

```diff
@@ -0,0 +1,63 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools
+
+MY_PN="dnssec-tools"
```

Comment:
This is thoroughly confusing. Kill those vars, as they are used unpredictably anyway.

Reply:
Yes. It sould be maybe questioned, if this package should not be killed entierly and integrated into ```net-dns/dnssec-tools```, since it included in the source of it. But no one replied to me, when I asked maintainer :-) What do you think in this case? It was splitted in past, but in current source it's included there.

Reply:
Start by removing the extra vars.

Reply:
Already done.